### ISSUE TITLE
send notification to generic webhook url

### ISSUE THREAD

**Issue by malesci** [farshadz1997/Microsoft-Rewards-bot#424]
Hello I see the possibility to send webhook to Discord URL, so I'm wondering if is possible or in case in you can implement the possibility to send notification to a generic webhook URL?

thanks

**Comment by AG4lyf** [comment]
there is `--send-to-webhook` that will send all kind of message to the webhook. is this what you are asking for ?

**Comment by AG4lyf** [comment]
or if you are asking for webhook other than discord. 
well you can pass it into discord flag. as the code is just a post request to that url. 
just need to change 2 lines on your end.

**Comment by malesci** [comment]
> or if you are asking for webhook other than discord.
> well you can pass it into discord flag. as the code is just a post request to that url.
> just need to change 2 lines on your end.

yes in fact I'm locking for webhook other than discord. At the end I would like to send webhook to some engine (maybe azure function or others), parse the message and sent notification to google spreadsheet that at the moment are unsupported from the script. 

so, thanks, I will try pass it into discord flag for this, thanks.
what are the 2 lines to change on my side?

thanks

**Comment by AG4lyf** [comment]
You have to change [these](https://github.com/farshadz1997/Microsoft-Rewards-bot/blob/5aa6d6afe91fdd2aed6694bb8220ee2a671ccb94/ms_rewards_farmer.py#L2253-L2255) lines. `message` is the message contains the content that is sent. you just need to change the `content` with the adequate headers your webhook accepts.
